### DISCUSSION TITLE
How relevent is today the info in wiki page "Issues with FPC, Lazarus"?

### DISCUSSION THREAD

**Discussion by ags1234** [doublecmd/doublecmd#1968] [General]
I was looking at the wiki page [Issues with FPC, Lazarus](https://github.com/doublecmd/doublecmd/wiki/Issues-with-FPC,-Lazarus) and I noticed it hasn't been updated since 2018.
It references older FPC versions.
Maybe things have changed since then, so are those points still relevant today?

**Comment by alexx2000** [comment]
1,2,3,5. Verification required.

4.. Still relevant today.

6,7. It's not important anymore, DC had been using the DWARF debugging format for a long time.

**Comment by ags1234** [comment]
Ok, thank you for the answer.
